magento/magento2
# New validation: 3bytes characters filter (4 bytes characters cannot be stored using UTF8)

Fixes #12058 

### Description
Added a new validation for regular emojis.

### Fixed Issues (if relevant)
1. magento/magento2#12058 : Can't save emoji in custom product options

### Manual testing scenarios
1. Add a Customizable Option to a product (text / textarea). 
2. Try to set an emoji into the text / textarea field in the frontend.
3. There should pop a validation error asking to remove emoji from field.

### Contribution checklist

<img width="1122" alt="captura de pantalla 2017-11-16 a las 16 32 35" src="https://user-images.githubusercontent.com/5971780/32899683-eccafa12-caeb-11e7-959b-524e9b49233e.png">
